workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/loadPath/FindMaxLoadPath.java

```diff
@@ -454,7 +461,12 @@ private UniformLoadInterval nextInterval() {
 				curStatusInterval.setStartUsers(curUsers);
 				curStatusInterval.setEndUsers(curUsers);
 				curStatusInterval.setDuration(getQosPeriodSec());
-				
+
+				statsIntervalComplete = true;
+				curStatsInterval.setName(nextInterval.getName());
+				curStatsInterval.setUsers(curUsers);
+				curStatsInterval.setDuration(getQosPeriodSec());
```

Comment:
name and duration set again.

Reply:
Same thing, curStatusInterval Vs curStatsInterval